toolset/benchmark/fortune_html_parser.py

```diff
@@ -74,20 +74,30 @@ def handle_charref(self, name):
     # Not sure why some are escaping '/'
     if val == "47" or val == "047" or val == "x2f":
       self.body.append("/")
+    # "&#40;" is a valid escaping of "(", but
+    # it is not required, so we need to normalize for out
+    # final parse and equality check.
+    if val == "40" or val == "040" or val == "x28":
```

Comment:
@jhass I just locally tested ffead-cpp and treefrog on the following branches:

- round-14
- add-missing-php-tests
- jhass/update_crystal

and am seeing Fortune pass there, so it's possible there's just something funky going on with Travis right now. I've also been seeing some weird MongoDB fails the past day or two. I'm still not sure what the issue is, but I'm good on merging your branch in now.